upstream-community-operators/myvirtualdirectory/myvirtualdirectory.1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,241 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: myvirtualdirectory.1.0.0
+  annotations:
```

Comment:
Please add the `metadata.annotations.capabilities` field following the maturity model that can be found at this link: https://www.operatorhub.io/getting-started#How-do-I-start-writing-an-Operator?
Please apply this change to both CSVs.

Reply:
Done